cmd/registry/cmd/root.go

```diff
@@ -57,6 +60,23 @@ func Command() *cobra.Command {
 	cmd.AddCommand(list.Command())
 	cmd.AddCommand(upload.Command())
 	cmd.AddCommand(vocabulary.Command())
+	cmd.AddCommand(RPCCommand())
+	return cmd
+}
+
+func RPCCommand() *cobra.Command {
```

Comment:
Can this be moved into `cmd/registry/cmd/rpc` too? I think it would provide a nicer boundary between code that we fully own/control, and code that requires knowledge of the GAPIC generator output.

Reply:
It could, but I am reluctant to do that because all the other go files in the `rpc` directory are automatically generated. I've left this as-is, or would be fine to put the rpc-related code in a separate file if you prefer.

Reply:
I have the same feeling of reluctance, wanting to have clear separation of the generated code from the handwritten integration code.

On the other hand, I have a similar feeling leading my suggestion to move this function. I want to have a clear separation of code that we control from code that relies on the generated commands. This feels more important to me because it affects where the errors will appear if the integration breaks on regen.

There's a middle path that might balance these desires: where we have package `cmd/registry/cmd/rpc` with a single file defining the `rpc` subcommand, and a second package holding the generated code. It would make for a very lightweight package, but I don't think there's anything wrong with that.

Reply:
I wouldn't strongly object to this in a future PR, but think we're talking about diminishing returns now, particularly since the new package would be <20 lines of code.